content/storage-history/history.textile

```diff
@@ -0,0 +1,596 @@
+---
+title: History
+meta_description: "Learn about accessing message history with the history and rewind features"
+languages:
+ - javascript 
+ - java 
+ - python 
+ - php 
+ - ruby 
+ - nodejs 
+ - objc
+ - swift
+ - go 
+ - csharp
+---
+
+History refers to accessing previously sent messages, enabling you to retrieve both channel history and presence history. If "storage":/storage-history/storage is enabled messages will typically be stored for 24 for free accounts or 72 hours for paid accounts, otherwise Ably retains the last two minutes of message history.
+
+h2(#history-rewind). History vs rewind
+
+There are two ways of accessing your channel history: history or rewind. Both history and rewind need the channel to have "persistence enabled":/storage-history/storage. 
+
+History is ideal for when you want to retrieve any message within your "storage time frame":/storage-history/storage other than the last published message, enabling a longer, more complete context. Rewind is useful for getting an instant update of a channel, enabling an update from either a given number of messages or a previous point in time as a time interval.
+
+History is useful for cases where you need to access messages published beyond the most recent, for example:
+
+* Chat applications where a longer period of chat history should be available to new joiners
+* Seeing the history of periodically sent data, such as weather apps that enable you to see recent weather 
+* IoT devices that care about recent statuses
+
+Whereas rewind would be useful in situations similar to the above, but with a more limited scope, for example:
+
+* Realtime chat applications that only need to show the most recent context, not an entire history
+* A channel providing periodic data, such as weather apps that only provide the immediate weather
+* IoT devices that only care about the current state
+
+Rewind is also the only way return the last event published to a channel persisted for 365 days if "persist last message is enabled":/storage-history/storage#persist-last-message.
+
+Note that rewind is only available with the realtime interface.
+
+h2(#retrieve-channel). Retrieve channel history
+
+You can retrieve channel history by using the "@history()@":/api/realtime-sdk/history#channel-history method. 
+
+The Ably SDKs provide a straightforward API to retrieve paginated messages or presence event history. By default each page of history contains up to 100 messages and is ordered from most recent to oldest.
+
+REALTIME-CODE
```

Comment:
Suggest adding a sentence introducing the example code and what it's doing here.